guides/v2.1/frontend-dev-guide/layouts/product-layouts.md

```diff
@@ -0,0 +1,43 @@
+---
+group: frontend-developer-guide
+title: Product layouts
+functional_areas:
+  - Frontend
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+This topic provides information about product layouts files. Magento allows to customize view pages for all product types in the common layout files or particular product types.
+
+## Product view page 
+ 
+Layout file | Description
+--- | ---
+`catalog_product_view.xml` | Common layout. Affects all product types
+`catalog_product_view_type_bundle.xml` | Layout directives from this file implements to `bundle` product only
```

Comment:
Please, change the "directive" term to the "layout" or "node". The directive term in development has a little bit different meaning and more applicable to an argument or a property. 
So in result we have something like:

"Layout from this file is applied to bundle products only" and so on.

Reply:
Good point, @rogyar !

Thank you!

**fixed**